src/eduid_common/api/app.py

```diff
@@ -100,6 +101,13 @@ def eduid_init_app_no_db(name, config, app_class=AuthnApp):
     app = init_sentry(app)
     app.session_interface = SessionFactory(app.config)
 
+    stats_hosts = app.config.get('STATS_HOSTS', False)
```

Comment:
I think we only support one stats host (correct me if I am wrong) so can we rename STATS_HOSTS to STATS_HOST?